zenchainlabs/profile.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "../profile.schema.json",
+  "name": "ZenChainLabs",
+  "identity": "C0586A29077BF9AE",
+  "website": "https://zenchainlabs.io",
+  "description": {
```

Comment:
This is a little undocumented still, but either remove the description or provide it as a set of keys similar to ecostake profile.json. This is invalid JSON currently

Reply:
Updated description json with key/value.
Thanks.